src/os/windows/mod.rs

```diff
@@ -186,16 +183,15 @@ impl Library {
     /// Additionally, the callers of this function must also ensure that execution of the
     /// termination routines contained within the library is safe as well. These routines may be
     /// executed when the library is unloaded.
-    pub unsafe fn load_with_flags<P: AsRef<OsStr>>(filename: P, flags: DWORD) -> Result<Library, crate::Error> {
+    pub unsafe fn load_with_flags<P: AsRef<OsStr>>(filename: P, flags: u32) -> Result<Library, crate::Error> {
```

Comment:
I’m not sure I’m particularly comfortable with this change. Ultimately `LoadLibraryExW` is accepting a DWORD and I have no way of ensuring that Microsoft won’t wake up one of these days and produce a Windows port with `DWORD != u32`. They have a long history of making some real difficult to understand choices after all.

`windows-sys` appears to have a [type alias](https://docs.rs/windows-sys/latest/windows_sys/Win32/System/LibraryLoader/type.LOAD_LIBRARY_FLAGS.html) for this argument. Is there a reason to not use that?

Reply:
fwiw, this is discussed a bit in the windows-rs repo: https://github.com/microsoft/windows-rs/issues/881 where they recommend to just use the underlying value.

Using the type alias is however a very good idea, done